platform/tests/common/mongodb.rs

```diff
@@ -10,38 +8,14 @@ use platform::mongodb::{Context, MongoDocument, Store};
 use platform::auth::*;
 use platform::mongodb::auth::DefaultAuthorizer;
 
-use crate::common::config_from_env;
-
-const USERNAME: &str = "harbor-test-admin";
-const PASSWORD: &str = "harbor-test-password";
 pub const DB_IDENTIFIER: &str = "harbor";
 pub const KEY_NAME: &str = "id";
 pub const COLLECTION: &str = "Group";
 
-pub struct ClusterContext {
-    endpoint: String,
-}
-
-impl ClusterContext {
-    #[allow(dead_code)]
-    pub fn connection_string(&self) -> String {
-        format!("mongodb://{}:{}@{}", USERNAME, PASSWORD, self.endpoint)
-    }
-}
-
-pub struct LocalContext;
-
-impl LocalContext {
-    #[allow(dead_code)]
-    pub fn connection_string() -> String {
-        "mongodb://localhost:27017".to_string()
-    }
-}
-
 #[allow(dead_code)]
 pub async fn local_context() -> Result<Context, Error> {
     let ctx = Context{
-        connection_uri: LocalContext::connection_string(),
+        connection_uri: "mongodb://localhost:27017".to_string(),
```

Comment:
is it possible to avoid hardcoding this?

Reply:
Yeah, I can do something similar to core.

FYI I am expecting a bit of this to change when we start integrating with CI.

Reply:
Sweet!